### PR TITLE
[Doc] Fix a typo in math/math_module.md

### PR DESCRIPTION
**Pull request by DavidXu-JJ**
Issue: The typo in the documentation is located at https://docs.taichi-lang.org/docs/math_module#small-vector-and-matrix-types

image

### Brief Summary

copilot:summary

### Walkthrough

copilot:walkthrough

### CODE REVIEW

**Comment by feisuzhu** [comment]
Could you please also fix the error in `Build and Test / Build and Test (Docs) (pull_request)` ?  It's unrelated to the typo, but the change forces this doc to be included in CI.

**Comment by DavidXu-JJ** [comment]
I'm trying to fix it. Let's wait for the test!

File: docs/lang/articles/math/math_module.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ u = v.rraa  # vec4(1, 1, 2, 2)
 - `ti.Vector` is a function that accepts a 1D array and returns a matrix instance that has only one column. For example, `ti.Vector([1, 2, 3, 4, 5])`.
 - `ti.types.vector` is a function that accepts an integer and a primitive type and returns a vector type. For example: `vec5f = ti.types.vector(5, float)`. `vec5f` can then be used to instantiate 5D vectors or annotate data types of function arguments and struct members:
     ```python
+    vec5f = ti.types.vector(5, float)
```
Comment (feisuzhu) [inline docs/lang/articles/math/math_module.md:116]:
```suggestion
    vec5f = ti.types.vector(5, float)
    
```

Comment (feisuzhu) [reply docs/lang/articles/math/math_module.md:116]: Empty line between type def and @ti.kernel

**Review by feisuzhu — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**